### PR TITLE
Executions empty state and table

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
JIRA:
[RHOAIENG-4776](https://issues.redhat.com/browse/RHOAIENG-4776)
[RHOAIENG-4777](https://issues.redhat.com/browse/RHOAIENG-4777)

## Description

Empty state:
Screenshot 2024-04-29 at 4 10 40 PM

Executions table:
Screenshot 2024-04-29 at 4 12 52 PM

## How Has This Been Tested?

1. Go to the executions page under Experiment navigation
2. Check the list
3. Check the paging and filtering

## Test Impact

N/A

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/pages/pipelines/global/experiments/executions/ExecutionsTableRowStatusIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          <ExclamationCircleIcon />
+        </Icon>
+      );
+      tooltip = 'Failed';
+      break;
+    case Execution.State.RUNNING:
+      icon = <Icon isInProgress />;
+      tooltip = 'Running';
+      break;
+    // TODO: change the icon here
```
Comment (Gkrumbach07) [inline frontend/src/pages/pipelines/global/experiments/executions/ExecutionsTableRowStatusIcon.tsx:56]: is this todo still valid

Comment (DaoDaoNoCode) [reply frontend/src/pages/pipelines/global/experiments/executions/ExecutionsTableRowStatusIcon.tsx:56]: Yes, I need to double-check with UX, thanks for the reminder. I can leave this comment here for now and update it later when I get the feedback from the UX.

**Comment by Gkrumbach07** [comment]
tested code and everything works

/lgtm
/approve

**Pull request merged**